LeaderboardBackend/Migrations/20240718220723_CategoryAddTimestamps.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+using NodaTime;
+
+#nullable disable
+
+namespace LeaderboardBackend.Migrations
+{
+    /// <inheritdoc />
+    public partial class CategoryAddTimestamps : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
In the future when data needs to persist, if EntityFramework has the ability to update column types, postgres is extremely good at coercing types for you in the majority of cases, so you don't need to drop tables. (this is one instance)

Reply:
These migrations are autogenerated. I did make sure to look at them though to make sure they were coercing where necessary.

Reply:
Be wary of a table drop for production code once things start populating -

Also a note to add db backups for prod :)

Reply:
We're not dropping here. This is adding columns. Dropping only occurs if you roll this migration back.

Reply:
OH

These are new columns - ignore me, and apologies. 

I'm not sure why i thought these had already existed. Sorry!